doc/subsystems/networking/qemu_setup.rst

```diff
@@ -133,18 +135,26 @@ To stop the daemons, press Ctrl+C in the corresponding terminal windows
 (you need to stop both ``loop-slip-tap.sh`` and ``loop-socat.sh``).
 
 
-Setting up Zephyr and NAT/masquerading on QEMU host to access Internet
-**********************************************************************
+.. _networking_internet:
 
-To access the internet from a Zephyr application using IPv4,
+Setting up Zephyr and NAT/masquerading on host to access Internet
+*****************************************************************
+
+To access the Internet from a Zephyr application, some additional
+setup on the host may be required. This setup is common for both
+application running in QEMU and on real hardware, assuming that
+a development board is connected to the development host (if a
```

Comment:
I'd jjust make this two sentences:

    ... connected to the development host.  If a
    board is connected to a dedicated router, this additional
    setup should not be needed.

Reply:
done